README.md

```diff
@@ -1,13 +1,56 @@
 # openghg_defs
 
-The path to the overall data path and primary definition JSON files are accessible using:
+This repository contains the supplementary information / metadata used by the OpenGHG project.
 
-```python
-import openghg_defs
+## Installation
 
-data_path = openghg_defs.data_path
+We recommend you install `openghg_defs` in the same virtual environment as OpenGHG.
 
-species_info_file = openghg_defs.species_info_file
-site_info_file = openghg_defs.site_info_file
-domain_info_file = openghg_defs.domain_info_file
+To install `openghg_defs` using `pip` run
+
+```bash
+pip install openghg_defs
+```
+
+## Usage
+
+The primary definition JSON files are accessible using either the loading functions
+
+- `load_site_info` - load site information dictionary
+- `load_domain_info` - load domain info dictionary
+- `load_species_info` - load species info dictionary
+
+For example
+
+```ipython3
+In [1]: import openghg_defs
+
+In [2]: species_info = openghg_defs.load_species_info()
+
+In [3]: species_info
+Out[3]:
+{'APO': {'alt': ['atmospheric_potential_oxygen'],
+  'group': 'Other',
+  'name': 'atmospheric_potential_oxygen',
+  'long_name': 'Atmospheric Potential Oxygen',
+  'mol_mass': 'None',
+  'print_string': 'APO',
+  'units': 'per meg'},
+ 'C2F6': {'alt': ['PFC-116', 'PFC116', 'hexafluoroethane'],
+  ...
+ ```
+
+You can also get the path of the file directly:
+
+```ipython3
+In [1]: from openghg_defs import get_datapath
+   ...:
+   ...: fpath = get_datapath(filename="site_info.json")
+
+In [2]: fpath
+Out[2]: PosixPath('/home/gareth/Documents/Devel/supplementary_data/openghg_defs/data/site_info.json')
 ```
+
+### Deprecated functionality
+
+Version 0.0.1 of `openghg_defs` exposed paths to the JSON files storing the metadata. These are now deprecated and will be removed in a future version.
```

Comment:
Will this be deprecated? We still need to include an alternative way to substitute a new path for this to be the case so perhaps we don't want to present this as strongly here.

Reply:
I would also add back the details here for how to access the paths even if it's not the primary way to do this anymore.

Reply:
Yeah I think they should be deprecated, we can just make sure people have moved over to them.

Reply:
But I'm not seeing an alternative to them for supplementing another file?

> The loading functions were updated to check for the passing in of a filepath, see: https://github.com/openghg/openghg/blob/5b68ba755b380e3aac0120d303bc8a7f5ce55994/openghg/util/_site.py#L8

This links to functionality in `openghg/openghg` not on this repository. If the wrappers are the new way to interact with this repository then we need a documented alternative in this repository for how to to do this.

Reply:
Yes or their own copy of the original file.

Reply:
I'm confused as to why that functionality needs to be added to this repo? If they want to create a copy in the same data folder in the repo they could use `get_datapath`, or they could just pass the path of the file to `get_site_info(site_filepath="/my/path.json"). Does that not cover the use cases we expect?

Reply:
Because this PR starts to move the file readers (and potentially the understanding of these types of files) to this repository. If this is being moved here, we should provide a consistent interface which allows for other files to be read in the same way.